pkg/common/connectionmanager/connectionmanager.go

```diff
@@ -111,6 +124,19 @@ func (cm *ConnectionManager) Logout() {
 	}
 }
 
+func (cm *ConnectionManager) Verify() error {
+	for vcServer := range cm.VsphereInstanceMap {
+		err := cm.Connect(context.Background(), vcServer)
```

Comment:
Consider passing `Context` as a param to `Verify()`, so there's an option to use a timeout for testing these connections.

Reply:
Good suggestion. Will provide both.